easybuild/easyconfigs/d/Dakota/Dakota_fix_boost.patch

```diff
@@ -0,0 +1,13 @@
+# author: Sam Moors
```

Comment:
please add the Dakota version to the patch name,
and add a short description to what the patch does, maybe with the error like this:
```
Fix CmakeLists.txt to avoid Boost error:
CMake Error at .../Boost/1.71.0-gompi-2019b/lib/cmake/Boost-1.71.0/BoostConfig.cmake:182 (boost_find_component):
  boost_find_component Macro invoked with incorrect arguments for macro
  named: boost_find_component
Call Stack (most recent call first):
  .../CMake/3.15.3-GCCcore-8.3.0/share/cmake-3.15/Modules/FindBoost.cmake:443 (find_package)
  CMakeLists.txt:363 (find_package)
```


and while you're at it, could you use my full name + affiliation? Samuel Moors (Vrije Universiteit Brussel)

Reply:
sure, sorry